examples/cms-sanity/README.md

```diff
@@ -0,0 +1,140 @@
+
+First, copy the `.env.example` file in this directory to `.env` (which will be ignored by Git):
+
+```bash
+cp .env.example .env
+```
+
+Then set each variable on `.env`:
+
+- `NEXT_EXAMPLE_CMS_SANITY_PREVIEW_SECRET` can be any random string (but avoid spaces), like `MY_SECRET` - this is used for [the Preview Mode](/docs/advanced-features/preview-mode.md).
+- `SANITY_PROJECT_ID` the `projectId` from the previous step
+
+Your `.env` file should look like this:
+
+```bash
+NEXT_EXAMPLE_CMS_SANITY_PREVIEW_SECRET=...
+SANITY_PROJECT_ID=...
+```
+
+### Step 4. Copy the schema file
+
+After initializing your Sanity studio project there should be a `schemas` folder.
+
+To get up and running we want to copy the schema file from [`./schemas/schema.js`](./schemas/schema.js) to the studio project to use for creating/editing content.
+
+### Step 5. Populate Content
+
+To add some content go to your Sanity studio project where we added the schema file and run `sanity start`.
+
+After the project has started and you have navigated to the URL given in the terminal, select **Author** and create a new record.
+
+- You just need **1 Author record**.
+- Use dummy data for the text.
+- For the image, you can download one from [Unsplash](https://unsplash.com/).
+
+Next, select **Post** and create a new record.
+
+- We recommend creating at least **2 Post records**.
+- Use dummy data for the text.
+- You can write markdown for the **Content** field.
+- For the images, you can download ones from [Unsplash](https://unsplash.com/).
+- Pick the **Author** you created earlier.
+
+**Important:** For each post record, you need to click **Publish** after saving. If not, the post will be in the draft state.
+
+### Step 6. Run Next.js in development mode
+
+```bash
+npm install
+npm run dev
+
+# or
+
+yarn install
+yarn dev
+```
+
+Your blog should be up and running on [http://localhost:3000](http://localhost:3000)! If it doesn't work, post on [GitHub discussions](https://github.com/zeit/next.js/discussions).
+
+### Step 7. Try preview mode
+
+On Sanity, go to one of the posts you've created and:
+
+- **Update the title**. For example, you can add `[Draft]` in front of the title.
+- As you edit the document it will be saved as a draft, but **DO NOT** click **Publish**. By doing this, the post will be in the draft state.
+
+Now, if you go to the post page on localhost, you won't see the updated title. However, if you use the **Preview Mode**, you'll be able to see the change ([Documentation](/docs/advanced-features/preview-mode.md)).
+
+To enable the Preview Mode, go to this URL:
+
+```
+http://localhost:3000/api/preview?secret=<secret>&slug=<slug>
+```
+
+- `<secret>` should be the string you entered for `NEXT_EXAMPLE_CMS_SANITY_PREVIEW_SECRET`.
+- `<slug>` should be the post's `slug` attribute (you can check on Sanity).
+
+You should now be able to see the updated title. To exit the preview mode, you can click **Click here to exit preview mode** at the top.
```

Comment:
Instead of this, we can have them use the Preview UI from the dashboard

![](https://cdn.sanity.io/images/3do82whm/next/1aad7fd94ab5565a9b4f4e21d4bbcd8ade025adf-511x327.png?w=1440&h=921&fit=clip&auto=format)

Reply:
Updated to mention this